.github/workflows/build_and_release.yml

```
@@ -0,0 +1,61 @@
+          npm i -g @vercel/ncc
+          npm i @actions/github
+          npm run build
+      - name: Commit
+        run: |
+          git config --global user.name 'yulin17-chang'
+          git config --global user.email '182771494+yulin17-chang@users.noreply.github.com'
+          git commit -am "dist build"
+          git push
+
+  release:
+    name : Release
```

yamllint

[warning] too many spaces before colon

(colons)